robot/EDA/tests/browser/contacts/preferred_phone.robot

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+
+Resource        robot/EDA/resources/EDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+Let Us Populate Create And Return Contact with Home Phone
+    [tags]          unstable
+    ${contact_id} =  Populate Create And Return Contact with Home Phone    Julian      Joseph   512-555-1234  
```

Comment:
Do you anticipate using this keyword in the future for other scripts ? We add keywords to EDA.robot file only if it is reusable across multiple tests. If this is only required for the test in question, I'd suggest you move the keyword code here in the script itself.

Reply:
@Safiyab thanks for the praise!

I imagine there would be other situations where we'd need to fill in the Phone, but I can move it inside the test if you think that's unlikely or if we should only do that if there's a clear use case. I could also make it a bit more flexible by allowing you to choose which Phone field you want to fill and/or allowing you to fill in multiple which I suspect would be more useful.

Reply:
I am not fully aware of the EDA features, but I just wondered how many cases would there be where you create a contact with just first/last name and home phone ? There might be more additional steps required, in which case this can't be reused.

Yeah you can make it flexible (only if you anticipate requiring that in the future), coz we don't want to complicate anything that might not be used later. Keep it simple for now, and if in future it needs an upgrade, this can be edited.

P.S: These are really good questions and ideas btw. We just keep refining as we go ! :)

Reply:
Thanks all make sense! I'll plan on just combining the "populate" keyboard into the existing test :-).

Reply:
@Safiyab moved the `Populate Create And Return Contact with Home Phone` keyword into the local test